flask/app/tasks.py

```diff
@@ -13,6 +13,9 @@ def send_email_notification(app):
         yesterday = (datetime.now() - timedelta(1)).strftime("%Y-%m-%d")
         analyses = (
             models.Analysis.query.options(
+                joinedload(models.Analysis.datasets).joinedload(
+                    models.Dataset.linked_files
+                ),
```

Comment:
Does this do it with two `joinedload(models.Analysis.datasets)` options. I had imaged that the remaining `joinedload(models.Dataset.tissue_sample)...` would be part of parameters to an `.options` call on this joinedload object along with linked_files. Let me check the final query...

Reply:
Yup, this looks good to me!